### DISCUSSION TITLE
为什么除了主页其他页面无法显示背景图片？

### DISCUSSION THREAD

**Discussion by xBLACKICEx** [BewlyBewly/BewlyBewly#292] [Q&A]
如题：除了主页其他页面均没有背景图片
主页：
![image](https://github.com/hakadao/BewlyBewly/assets/51036094/5060c33c-996d-45f7-b6dd-93807e235a33)
视频页：
![image](https://github.com/hakadao/BewlyBewly/assets/51036094/1abd9c1a-d54d-4e0b-aa7c-bcc0a2295d39)
文章页：
![image](https://github.com/hakadao/BewlyBewly/assets/51036094/650c2232-811c-4e50-afd4-71c695d67ede)
搜索页：
![image](https://github.com/hakadao/BewlyBewly/assets/51036094/76f25bd9-611f-4dc4-ae1c-2a049a068905)

**Comment by hakadao** [comment]
目前只對首頁有效果，其他頁面不作調整因為後期可能會做成彈窗效果（兩邊設定背景有點多餘了）
